tests/pattern_rewriter_test.py

```diff
@@ -16,9 +16,9 @@
 def rewrite_and_compare(prog: str, expected_prog: str,
                         walker: PatternRewriteWalker):
     ctx = MLContext()
-    builtin = Builtin(ctx)
-    arith = Arith(ctx)
-    scf = Scf(ctx)
+    builtin = Builtin(ctx)  # noqa
```

Comment:
If we are not using them, we can either write `_ = Builtin(ctx)`, or `Builtin(ctx)`

Reply:
right